src/components/Grid/Grid.tsx

```diff
@@ -95,9 +94,9 @@ export const Grid: FC<GridProps> = ({
 }) => {
   const [size, setSize] = useState<Size | null>(null);
   const [items, setItems] = useState(initialItems);
-  const [classicArrowItems, setClassicArrowItems] = useState<
-    Array<ClassicArrowItemType>
-  >((initialArrowItems as ClassicArrowItemType[]) ?? []);
+  const [ArrowItems, setArrowItems] = useState<Array<ArrowItemType>>(
+    (initialArrowItems as ArrowItemType[]) ?? [],
+  );
```

Comment:
Don't capitalize `arrowItems`. Is the type cast still necessary?
```suggestion
  const [ArrowItems, setArrowItems] = useState<Array<ArrowItemType>>(
    initialArrowItems ?? [],
  );
```